modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc

```diff
@@ -0,0 +1,198 @@
+// This module is included in the following assembly:
+//
+// */cicd/pipelines/setting-compute-resource-quota-for-openshift-pipelines.adoc
+
+
+[id="specifying-pipelines-resource-quota-using-priority-class_{context}"]
+= Specifying pipelines resource quota using priority class
+
+A `PriorityClass` object defines a mapping from a priority class name to the integer value of the priority, with higher value indicating higher priority. After you create a priority class, you can create pods that specify the priority class name in their specifications. In addition, you can control a pod's consumption of system resources based on the pod's priority.
```

Comment:
```suggestion
A `PriorityClass` object maps priority class names to the integer values that indicates their relative priorities. Higher values grant higher priority to a class. After you create a priority class, you can create pods that specify the priority class name in their specifications. In addition, you can control a pod's consumption of system resources based on the pod's priority.
```
This suggestion is based on my guess that object maps multiple class names to multiple priorities.